src/request/remove-key/index.html

```diff
@@ -72,28 +74,99 @@
+                        <path opacity="0.78" d="M39.0519 54.7279L37.1217 51.3821C36.984 51.1428 36.7279 50.9961 36.4525 50.9961H32.5921C32.3167 50.9961 32.0619 51.1441 31.9242 51.3821L29.9927 54.7279C29.855 54.9672 29.855 55.2606 29.9927 55.5L31.9229 58.8457C32.0606 59.0851 32.3167 59.2318 32.5921 59.2318H36.4525C36.7279 59.2318 36.9827 59.0838 37.1204 58.8457L39.0506 55.5C39.1896 55.2619 39.1896 54.9672 39.0519 54.7279Z" fill="white"/>
+                        <path opacity="0.5" d="M30.6291 40.1302L28.6988 36.7845C28.5611 36.5451 28.3051 36.3984 28.0297 36.3984H24.1692C23.8938 36.3984 23.639 36.5464 23.5013 36.7845L21.5698 40.1302C21.4321 40.3696 21.4321 40.663 21.5698 40.9023L23.5001 44.2481C23.6377 44.4874 23.8938 44.6341 24.1692 44.6341H28.0297C28.3051 44.6341 28.5599 44.4861 28.6975 44.2481L30.6278 40.9023C30.7668 40.6643 30.7668 40.3696 30.6291 40.1302Z" fill="white"/>
+                        <path opacity="0.6" d="M30.6291 49.8607L28.6988 46.515C28.5611 46.2756 28.3051 46.1289 28.0297 46.1289H24.1692C23.8938 46.1289 23.639 46.2769 23.5013 46.515L21.5698 49.8607C21.4321 50.1001 21.4321 50.3934 21.5698 50.6328L23.5001 53.9785C23.6377 54.2179 23.8938 54.3646 24.1692 54.3646H28.0297C28.3051 54.3646 28.5599 54.2166 28.6975 53.9785L30.6278 50.6328C30.7668 50.3947 30.7668 50.1001 30.6291 49.8607Z" fill="white"/>
+                        <path opacity="0.8" d="M47.4748 40.1302L45.5445 36.7845C45.4068 36.5451 45.1508 36.3984 44.8754 36.3984H41.0149C40.7395 36.3984 40.4847 36.5464 40.347 36.7845L38.4155 40.1302C38.2778 40.3696 38.2778 40.663 38.4155 40.9023L40.3458 44.2481C40.4835 44.4874 40.7395 44.6341 41.0149 44.6341H44.8754C45.1508 44.6341 45.4056 44.4861 45.5433 44.2481L47.4735 40.9023C47.6125 40.6643 47.6125 40.3696 47.4748 40.1302Z" fill="white"/>
+                        <path opacity="0.6" d="M47.4743 49.8607L45.544 46.515C45.4064 46.2756 45.1503 46.1289 44.8749 46.1289H41.0144C40.739 46.1289 40.4843 46.2769 40.3466 46.515L38.415 49.8607C38.2773 50.1001 38.2773 50.3934 38.415 50.6328L40.3453 53.9785C40.483 54.2179 40.739 54.3646 41.0144 54.3646H44.8749C45.1503 54.3646 45.4051 54.2166 45.5428 53.9785L47.473 50.6328C47.612 50.3947 47.612 50.1001 47.4743 49.8607Z" fill="white"/>
+                        </g>
+                        <defs>
+                        <radialGradient id="paint0_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(68.6666 116.277) rotate(-180) scale(67.3333 114.941)">
+                        <stop stop-color="#FD6216"/>
+                        <stop offset="1" stop-color="#FC8701"/>
+                        </radialGradient>
+                        <clipPath id="clip0">
+                        <rect width="44.125" height="5.33333" fill="white" transform="translate(12.8889 9.55469)"/>
+                        </clipPath>
+                        </defs>
+                        </svg>
```

Comment:
Here is a minified version of this SVG:
```svg
<svg width="70" height="118" fill="none" xmlns="http://www.w3.org/2000/svg"><rect width="70" height="117.6" rx="3.4" fill="#fff"/><path d="M1.3 3.6c0-1.2 1-2.3 2.3-2.3h62.8c1.2 0 2.3 1 2.3 2.3V114c0 1.3-1 2.3-2.3 2.3H3.6c-1.2 0-2.3-1-2.3-2.3V3.6z" fill="url(#paint0_radial)"/><g opacity=".5" fill="#fff"><g clip-path="url(#clip0)"><path d="M18.8 12l-1.2-2.2c-.1-.2-.3-.2-.5-.2h-2.5c-.1 0-.3 0-.4.2L13 12v.5l1.2 2.1c.1.2.3.3.4.3h2.5c.2 0 .4-.1.5-.3l1.2-2.1V12z"/><rect x="20.5" y="10.3" width="14.9" height="3.7" rx="1.9"/><rect x="37.3" y="10.3" width="18.7" height="3.7" rx="1.9"/></g><rect x="26.1" y="19.4" width="17.7" height="1.9" rx=".9"/></g><g opacity=".5" fill="#fff"><path fill-rule="evenodd" clip-rule="evenodd" d="M28.2 81.6H20c-.7 0-1.3-.6-1.3-1.3V72c0-.8.6-1.4 1.3-1.4h8.2c.7 0 1.4.6 1.4 1.4v8.2c0 .7-.7 1.3-1.4 1.3zm-6.5-8.1c-.2 0-.3.1-.3.3v4.8c0 .1.1.3.3.3h4.8c.2 0 .3-.2.3-.3v-4.8c0-.2-.1-.3-.3-.3h-4.8zM20 92.5h8.2c.7 0 1.4.6 1.4 1.4v8.1c0 .8-.7 1.4-1.4 1.4H20c-.7 0-1.3-.6-1.3-1.4V94c0-.8.6-1.4 1.3-1.4zm6.5 8.2c.2 0 .3-.2.3-.4v-4.7c0-.2-.1-.4-.3-.4h-4.8c-.2 0-.3.2-.3.4v4.7c0 .2.1.4.3.4h4.8zM41.8 70.7H50c.7 0 1.3.6 1.3 1.4v8.2c0 .7-.6 1.3-1.3 1.3h-8.2c-.7 0-1.4-.6-1.4-1.3V72c0-.8.7-1.4 1.4-1.4zm6.5 8.2c.2 0 .3-.2.3-.3v-4.8c0-.2-.1-.3-.3-.3h-4.8c-.2 0-.3.1-.3.3v4.8c0 .1.1.3.3.3h4.8z"/><path d="M32.3 76.5h1.3a1 1 0 0 0 0-2 .3.3 0 0 1-.3-.4v-2a1 1 0 1 0-2 0v3.4c0 .6.4 1 1 1zM36.4 73.1c.1 0 .3.2.3.4v7.4a1 1 0 1 0 2 0v-8.8c0-.6-.4-1-1-1h-1.3a1 1 0 0 0 0 2zM26.5 85c0 .6.4 1 1 1h4.8c.5 0 1-.4 1-1v-5.4a1 1 0 0 0-2 0v4c0 .3-.2.4-.4.4h-3.4a1 1 0 0 0-1 1z"/><path d="M22.7 84a1 1 0 0 0-1 1v3.4c0 .2-.1.4-.3.4H20a1 1 0 1 0 0 2h17.7c.6 0 1-.4 1-1v-4a1 1 0 0 0-2 0v2.6c0 .2-.2.4-.3.4H24a.3.3 0 0 1-.3-.4V85c0-.5-.5-1-1-1zM37.4 93.9c0-.6-.5-1-1-1h-4.1a1 1 0 0 0-1 1v5.4a1 1 0 1 0 2 0v-4c0-.3.2-.4.3-.4h2.8c.5 0 1-.5 1-1zM50 101H38.4a.3.3 0 0 1-.3-.3V98a1 1 0 1 0-2 0v4c0 .6.4 1 1 1H50a1 1 0 1 0 0-2z"/><path fill-rule="evenodd" clip-rule="evenodd" d="M45.2 98.3h-4a1 1 0 0 1-1.1-1v-4.1c0-.6.5-1 1-1h4.1c.6 0 1 .4 1 1v4c0 .6-.4 1.1-1 1.1zm-2.7-4c-.2 0-.4 0-.4.3v1.3c0 .2.2.4.4.4h1.3c.2 0 .4-.2.4-.4v-1.3c0-.2-.2-.4-.4-.4h-1.3z"/><path d="M49.3 88a1 1 0 0 0-1 1.1V98a1 1 0 1 0 2 0V89c0-.6-.4-1-1-1zM50.3 84.3c0-.5-.4-1-1-1h-7.5a1 1 0 0 0-1 1v4.1a1 1 0 1 0 2 0v-2.7c0-.2.2-.3.4-.3h6c.7 0 1.1-.5 1.1-1z"/></g><g opacity=".3" fill="#fff"><path opacity=".7" d="M39 35.3l-1.9-3.4a.8.8 0 0 0-.6-.4h-4c-.2 0-.4.2-.6.4l-2 3.4v.7l2 3.4c.2.2.4.4.7.4h3.9c.2 0 .5-.2.6-.4l2-3.4v-.7z"/><path opacity=".8" d="M39 54.7l-1.9-3.3a.8.8 0 0 0-.6-.4h-4c-.2 0-.4.1-.6.4l-2 3.3v.8l2 3.3c.2.3.4.4.7.4h3.9c.2 0 .5-.1.6-.4l2-3.3v-.8z"/><path opacity=".5" d="M30.6 40.1l-1.9-3.3a.8.8 0 0 0-.7-.4h-3.8c-.3 0-.6.1-.7.4l-2 3.3v.8l2 3.3c.1.3.4.4.7.4H28c.3 0 .6-.1.7-.4l2-3.3v-.8z"/><path opacity=".6" d="M30.6 49.9l-1.9-3.4a.8.8 0 0 0-.7-.4h-3.8c-.3 0-.6.2-.7.4l-2 3.4v.7l2 3.4c.1.2.4.4.7.4H28c.3 0 .6-.2.7-.4l2-3.4V50z"/><path opacity=".8" d="M47.5 40.1l-2-3.3a.8.8 0 0 0-.6-.4H41c-.3 0-.5.1-.7.4L38.4 40c-.1.3-.1.6 0 .8l2 3.3c0 .3.3.4.6.4H45c.3 0 .5-.1.6-.4l2-3.3c.1-.2.1-.5 0-.8z"/><path opacity=".6" d="M47.5 49.9l-2-3.4a.8.8 0 0 0-.6-.4H41c-.3 0-.5.2-.7.4L38.4 50v.7l2 3.4c0 .2.3.4.6.4H45c.3 0 .5-.2.6-.4l2-3.4V50z"/></g><defs><radialGradient id="paint0_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(-67.3333 0 0 -114.941 68.7 116.3)"><stop stop-color="#FD6216"/><stop offset="1" stop-color="#FC8701"/></radialGradient><clipPath id="clip0"><path fill="#fff" transform="translate(12.9 9.6)" d="M0 0h44.1v5.3H0z"/></clipPath></defs></svg>
```
Please also take care of indentation.